### PR TITLE
Add it-IT language

### PR DESCRIPTION
**Pull request by imcatta**
### Context
Added italian translations

### How has this been tested?
No tests needed

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Additional language file or change to the existing one (translations)

### Related issue(s):
#4696

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
Thank you for sharing the translation and signing CLA @imcatta

**Comment by AMBudnik** [comment]
I have [created a topic](https://github.com/handsontable/handsontable/issues/5407 ) in which we gather confirmations for the translation. We need 5 of them.

**Comment by AMBudnik** [comment]
I got one confirmation from our new user 👍

**Comment by AMBudnik** [comment]
I got a second confirmation 👍

**Comment by AMBudnik** [comment]
We got a 3 confirmation, but it was a suggestion to change `riga padre` to `riga madre` but both should be correct. Please share your opinion @imcatta

**Comment by AMBudnik** [comment]
And 4th confirmation from @giovannipessiva at https://github.com/handsontable/handsontable/issues/5407 👍 

Great! We need one more and we can add Italian to the official language pack.

**Comment by imcatta** [comment]
> We got a 3 confirmation, but it was a suggestion to change `riga padre` to `riga madre` but both should be correct. Please share your opinion @imcatta

I agree, it's better to say `Scollega da riga madre` since `riga` is a female word.
@AMBudnik How can i fix this? It's my first pull request :)

**Comment by AMBudnik** [comment]
@imcatta already done, the Github UI allows you to click the 'edit' button on the already published file.

**Comment by DenialOfLupsy** [comment]
The change addresses #3 but job fails for dependencies issues:
```
$ GOOS=windows go build -o duckdns-windows
../../sirupsen/logrus/terminal_windows.go:10:2: cannot find package "github.com/konsorten/go-windows-terminal-sequences" in any of:
	/home/travis/.gimme/versions/go1.8.linux.amd64/src/github.com/konsorten/go-windows-terminal-sequences (from $GOROOT)
	/home/travis/gopath/src/github.com/konsorten/go-windows-terminal-sequences (from $GOPATH)
The command "GOOS=windows go build -o duckdns-windows" exited with 1.
```

**Comment by wojciechczerniak** [comment]
We've got a Code Freeze on Wednesday. @imcatta If you manage to resolve latest review from @DenialOfLupsy before it that would be our 5th confirmation, I think. We would be happy to merge it and publish with the upcoming release next week. Great job all!

File: src/i18n/languages/it-IT.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  [C.FILTERS_LABELS_DISJUNCTION]: 'O',
+
+  [C.FILTERS_BUTTONS_SELECT_ALL]: 'Seleziona tutto',
+  [C.FILTERS_BUTTONS_CLEAR]: 'Pulisci',
+  [C.FILTERS_BUTTONS_OK]: 'OK',
+  [C.FILTERS_BUTTONS_CANCEL]: 'Cancella',
+
+  [C.FILTERS_BUTTONS_PLACEHOLDER_SEARCH]: 'Cerca',
+  [C.FILTERS_BUTTONS_PLACEHOLDER_VALUE]: 'Valore',
+  [C.FILTERS_BUTTONS_PLACEHOLDER_SECOND_VALUE]: 'Secondo valore'
```
[the remark this answers is not in the document]
Comment (imcatta) [reply src/i18n/languages/it-IT.js]: You're right, thanks!

**Comment by AMBudnik** [comment]
Thanks for merging! :)

**Comment by AMBudnik** [comment]
Thank you very much for sharing the translation and checking it

Italian is now the official language of Handsontable Pro 6.1.0+

**Comment by adrianszymanski89** [comment]
Hi @imcatta1. We just announced an introduction of a new functionality called `Copy with headers`. Could you help us to translate three short lines for the context menu?

**Pull request merged**